root/usr/lib/systemd/system/asterisk.service

```diff
@@ -14,7 +14,10 @@ RuntimeDirectoryMode=0775
 User=asterisk
 Group=asterisk
 ExecStart=/usr/sbin/asterisk -f -C /etc/asterisk/asterisk.conf
+ExecStartPost=/bin/bash -c "while ! /usr/sbin/asterisk -rx 'core show version' &>/dev/null; do sleep 1; ((++attempt<30)) || exit 2; done"
+ExecStartPost=/bin/bash -c "while [[ $$(/usr/sbin/asterisk -rx 'core show version') != Asterisk* ]]; do sleep 1; ((++attempt<30)) || exit 2; done"
 ExecStop=/usr/sbin/asterisk -rx 'core stop now'
+ExecStopPost=/bin/bash -c "while /usr/sbin/asterisk -rx 'core show version' &>/dev/null; do sleep 1; ((++attempt<30)) || exit 2; done"
```

Comment:
The issue is on the starting part: why we need also an extra `ExecStopPost`? 
It doesn't matter if asterisk takes seconds to shutdown.